dotcom-rendering/src/web/server/showMoreCardsToHtml.tsx

```diff
@@ -0,0 +1,27 @@
+import { renderToString } from 'react-dom/server';
+import { ExtraCardsContainer } from '../components/ExtraCardsContainer';
+
+interface DCRShowMoreContainer {
+	cards: DCRFrontCard[];
+	containerPalette?: DCRContainerPalette;
+}
+/**
+ * cardsToHtml is used by the /Cards endpoint to render extra fronts cards
+ *
+ * @returns string (the html)
+ */
+
+export const cardsToHtml = ({
```

Comment:
If the export of the file is `cardsToHtml` the name of the file should be the same